flaskr_tests.py

```diff
@@ -62,14 +81,58 @@ def test_edit(self):
             follow_redirects=True)
         self.app.get('/entry=1', follow_redirects=True)
         rv = self.app.post(
-            '/entry=1', 
-            data = dict(title='Bye',
-                        text='Nothing is allowed here'),
+            '/entry=1',
+            data=dict(title='Bye',
+                      text='Nothing is allowed here'),
             follow_redirects=True)
         assert b'HTML' not in rv.data
         assert b'Entry was successfuly edited' in rv.data
         assert b'Bye' in rv.data
 
+    def test_delete(self):
+        """Test entry delete."""
+        self.login('admin', 'default')
```

Comment:
There is same setup code here and in the 'test_edit' which means that these methods need to be moved to the separate test class and the common setup should be done in the setUp method.

The common code, related to all DB tests can be moved higher up, into the DBTestCase class, you can put current setUp / tearDown and login / logout helpers in there.